docs/presentation/Content/PropertyEditorSpecification.md

```diff
@@ -1,69 +1,25 @@
-# PropertyEditor Specification
+# Property Editor Specification
```

Comment:
Got anything specific in mind? 

Currently it's linked from the description of the only attribute using this type. IMO that should be enough, since this specification can't be used on its own.

Reply:
Nevermind, I did not realize this page was linked by PropertySpecification page.